Makefile

```diff
@@ -18,7 +18,10 @@ clean: ## remove files created during build
 	rm -rf vendor
 	rm -f coverage.*
 	rm -f cover.out
+	rm -rf *.log
 	rm -rf **/*.log
+	rm -f results.*
```

Comment:
If we use make, it will delete all logs and results named files?

Reply:
all logs in all subdirectories, and only results in the root directory

Reply:
We really want this? I'm concerned, because, maybe, we want to preserve those files for future debugging or checking, no?

Reply:
then you should not run make clean